src/components/History/Widget/WidgetCardWithdraw.tsx

```diff
@@ -18,6 +19,10 @@ type Props = {
 
 const NEAR_EXPLORER = process?.env?.nearExplorer ?? ""
 
+const nearToken = LIST_NETWORKS_TOKENS.find(
+  (token) => token.address === "wrap.near"
```

Comment:
Maybe we put nearToken in a constant to avoid calling find ? Same thing could be done as for tokens list

```
export const LIST_NETWORKS_TOKENS: NetworkToken[] =
  environment === "development"
    ? listNetworksTokensTestnet
    : listNetworksTokensMainnet
```

Reply:
We also insert a constant in the list for both mainnet and testnet

Reply:
We will not have a list of a thousand tokens so I believe it will not give us much capacity, but you are right, we use that date in several places so I'll do update in next commit.